pylint/checkers/base.py

```diff
@@ -1316,19 +1427,13 @@ def _find_name_group(self, node_type):
         return self._name_group.get(node_type, node_type)
 
     def _raise_name_warning(self, node, node_type, name, confidence):
-        type_label = _NAME_TYPES[node_type][1]
-
-        hint_name = node_type.replace('_', '-') + '-name-hint'
-        hint_part = '%r template' % hint_name
-
-        if self.config.include_naming_hint:
-            hint_rgx = getattr(self.config, hint_name.replace('-', '_'))
-            hint_part += ' (hint: %r)' % hint_rgx
+        type_label = HUMAN_READABLE_TYPES[node_type]
```

Comment:
I would still keep the naming hint flag (``include-naming-hint``) but only for naming styles, since the pattern already covers this. I'm thinking this would be a bit more helpful since there is no other way to see what we decided for a style to contain other than looking over the code (and the values won't be in the configuration either), so ``--include-naming-hint`` would be an user's only chance of seeing what a style value represents.